integration_tests/cellarfees_test.go

```diff
@@ -0,0 +1,378 @@
+		kb, err := val.keyring()
+		s.Require().NoError(err)
+
+		ethClient, err := ethclient.Dial(fmt.Sprintf("http://%s", s.ethResource.GetHostPort("8545/tcp")))
+		s.Require().NoError(err)
+
+		clientCtx, err := s.chain.clientContext("tcp://localhost:26657", &kb, "val", val.keyInfo.GetAddress())
+		s.Require().NoError(err)
+
+		auctionQueryClient := auctiontypes.NewQueryClient(clientCtx)
+		bankQueryClient := banktypes.NewQueryClient(clientCtx)
+		cellarfeesQueryClient := cellarfeestypes.NewQueryClient(clientCtx)
+		corkQueryClient := corktypes.NewQueryClient(clientCtx)
+		distQueryClient := disttypes.NewQueryClient(clientCtx)
+
+		s.T().Log("Verify that the first validator address is an approved cellar ID")
+		idsRes, err := corkQueryClient.QueryCellarIDs(context.Background(), &corktypes.QueryCellarIDsRequest{})
+		s.Require().NoError(err)
+
+		var found bool
+		for _, id := range idsRes.CellarIds {
+			if id == ethereumSender {
+				found = true
+				break
+			}
+		}
+		s.Require().True(found, "validator ethereum address %s is not an approved cellar ID", ethereumSender)
+
+		s.T().Logf("Verify that the module account's fee balances are zero")
+		ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+		defer cancel()
+
+		acctsRes, err := cellarfeesQueryClient.QueryModuleAccounts(ctx, &cellarfeestypes.QueryModuleAccountsRequest{})
+		s.Require().NoError(err, "Failed to query module accounts")
+
+		feesAddress := acctsRes.FeesAddress
+		s.T().Logf("Fees address: %s", feesAddress)
+		balanceRes, err := bankQueryClient.Balance(ctx, &banktypes.QueryBalanceRequest{
+			Address: feesAddress,
+			Denom:   fmt.Sprintf("gravity%s", alphaERC20Contract.Hex()),
+		})
+		s.Require().NoError(err, "Failed to query fee balance of denom %s", alphaERC20Contract.Hex())
+		s.Require().Zero(balanceRes.Balance.Amount.Uint64())
+		balanceRes, err = bankQueryClient.Balance(ctx, &banktypes.QueryBalanceRequest{
+			Address: feesAddress,
+			Denom:   fmt.Sprintf("gravity%s", betaERC20Contract.Hex()),
+		})
+		s.Require().NoError(err, "Failed to query fee balance of denom %s", betaERC20Contract.Hex())
+		s.Require().Zero(balanceRes.Balance.Amount.Uint64())
+
+		s.T().Log("Getting baseline rewards increase rate")
+		rewardsRes, err := distQueryClient.DelegationRewards(ctx, &disttypes.QueryDelegationRewardsRequest{
+			DelegatorAddress: val.keyInfo.GetAddress().String(),
+			ValidatorAddress: "sommvaloper199sjfhaw3hempwzljw0lgwsm9kk6r8e5ef3hmp",
+		})
+		s.Require().NoError(err)
+		startAmount := rewardsRes.Rewards.AmountOf(testDenom)
+		time.Sleep(time.Second * 12)
+		rewardsRes, err = distQueryClient.DelegationRewards(ctx, &disttypes.QueryDelegationRewardsRequest{
+			DelegatorAddress: val.keyInfo.GetAddress().String(),
+			ValidatorAddress: "sommvaloper199sjfhaw3hempwzljw0lgwsm9kk6r8e5ef3hmp",
+		})
+		s.Require().NoError(err)
+		endAmount := rewardsRes.Rewards.AmountOf(testDenom)
+		rewardRateBaseline := endAmount.Sub(startAmount).Quo(sdk.NewDec(12))
```

Comment:
Now that inflation has been disabled, do you still need all these baseline calculations or can we just delete it and test for fixed values?